cfpb/cfgov-refresh
# Bump regulations-site to 2.2.2

This commit bumps the optional regulations-site package to 2.2.2. That release adds a notification to the sidebar of all pages in every regulation that's identified as having an update in progress in the `EREGS_REGULATION_UPDATES` Django setting. See cfpb/regulations-site#850 for details on the changes to `regulations-site`.

## Changes

- Bump `regulations-site` to 2.2.2

## Testing

1. Install regulations-site 2.2.2
2. Run a local server with `EREGS_API_BASE=https://www.consumerfinance.gov/eregs-api/ ./runserver.sh`
3. Pop on over to eRegs at http://localhost:8000/eregulations/
4. View any of the regs that have an update in progress: 1002, 1010, 1011, 1012, 1013, 1024, or 1026. The various pages of those regs should look like the screenshots below.
6. The link in the notification should point to https://www.consumerfinance.gov/policy-compliance/rulemaking/final-rules/
7. Visit a couple of sections in a regulation with the notification to make sure the notification only appears once in the sidebar.
8. View any of the other regs that don't have an update in progress: 1004 or 1030. There should be no notification in the sidebar on any pages of those regs.

## Screenshots

Section with no SxS | Section with SxS | Section with definition | SxS | Diff | Search
------------------- | ---------------- | ----------------------- | --- | ---- | ------
![section-no-sxs](https://user-images.githubusercontent.com/1862695/38639981-e71274e2-3da0-11e8-8836-3d73152e9172.png) | ![section-yes-sxs](https://user-images.githubusercontent.com/1862695/38639986-eaf67ffe-3da0-11e8-9618-7dba18d4afdf.png) | ![definition](https://user-images.githubusercontent.com/1862695/38639999-eed9a6a0-3da0-11e8-8eea-231d17a9398b.png) | ![sxs](https://user-images.githubusercontent.com/1862695/38640009-f2d466a0-3da0-11e8-8654-35702f4c4036.png) | ![diff](https://user-images.githubusercontent.com/1862695/38640015-f6ea1ce4-3da0-11e8-9c07-3929e62f6077.png) | ![search](https://user-images.githubusercontent.com/1862695/38640021-faf072fc-3da0-11e8-8f28-65f0fc1b5f31.png)

## Notes

- As seen in the "reg with definition" screenshot above, definitions will appear on top of the alert. That's intentional but not ideal (always keeping the alert on top of definitions would have required too many changes for the scope of this PR).
- The alert doesn't consistently appear on the search results page, but that's a limitation in the search results page template and not with the alert. Given that you'll still see the alert on any page you get to from the search results page, this is again intentional.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
